app/components/LoadingButton/LoadingButton.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * LoadingButton.js
+ *
+ * Wraps the loading indicator in a tag with the .btn--loading class
+ */
+
+import React from 'react'
+import LoadingIndicator from '../../components/LoadingIndicator/LoadingIndicator'
+
+function LoadingButton(props) {
```

Comment:
are we doing es6 per FE meeting?

Reply:
We should wait until we finalise the Airbnb style rules we will follow